Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "eigen-client"
+description = "Eigen Client"
+version = "0.1.0"
```

Comment:
Is this stable enough ot be 0.1.0 or should we keep 0.0.1 for now?

Reply:
Nvm! semver recommends starting at 0.1.0: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase
Resolving. :)